BackEnd/Core/sailfish-common/build.gradle

```diff
@@ -50,13 +60,32 @@ dependencies {
 
 task templates {
     doLast {
-	    ant.taskdef(name: 'jamon', classname: 'org.jamon.ant.JamonTask',   classpath: configurations.jamon.asPath)
-	    ant.taskdef(name: 'xjc',   classname: 'com.sun.tools.xjc.XJCTask', classpath: configurations.jaxb.asPath)
-	    ant.jamon(destdir: "${genJavaDir}", srcdir: "${templatesDir}")
-        ant.xjc(schema: "${xsdDir}/dictionary.xsd", destdir: "${genJavaDir}", package: "com.exactpro.sf.common.impl.messages.xml.configuration", removeOldOutput: "yes") {
-            produces(dir: "${genJavaDir}/com/exactpro/common/sf/impl/messages/xml/configuration", includes: "*")
-	        arg(value: "-extension")
-	    }
+        ant.taskdef(name: 'jamon', classname: 'org.jamon.ant.JamonTask',   classpath: configurations.jamon.asPath)
+        ant.taskdef(name: 'xjc',   classname: 'com.sun.tools.xjc.XJCTask', classpath: configurations.jaxb.asPath)
+        ant.jamon(destdir: "${genJavaDir}", srcdir: "${templatesDir}")
+        ant.xjc(schema: "${xsdDir}/dictionary.xsd", destdir: "${genJavaDir}", package: "com.exactpro.sf.common.impl.messages.xml.configuration") {
+            produces(dir: "${genJavaDir}/com/exactpro/sf/common/impl/messages/xml/configuration", includes: "*")
+            arg(value: "-extension")
+        }
+    }
+}
+
+task schemaGen () {
```

Comment:
We should add a dependency on the **jar** task because of project artifacts in the classpath.